landroidcc/__init__.py

```
@@ -253,6 +355,53 @@ def _api_authentificate(self, username, password):
+    def _api_authentificate(self, username, password):
+        post_json = {
+            "username": username,
+            "password": password,
+            "grant_type": "password",
+            "client_id": self.WORX_CLIENT_ID, # Updated
+            "type": "app",
+            "client_secret": "nCH3A0WvMYn66vGorjSrnGZ2YtjQWDiCvjg7jNxK",
+            "scope": "*"
+        }
+        # Direct call to the new AUTH_URL instead of the v2 API path
+        response_plain = requests.post(self.AUTH_URL, json=post_json)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)